### PR TITLE
fix(bug): Some fixes in LicenseInfo DOCX generation

### PR DESCRIPTION
**Pull request by bs-jokri**
https://github.com/sw360/sw360portal/issues/575

* License names have LicenseRef prefix in License Info Documents when using SPDX
* License Texts miss formatting in lLicense Info document when using SPDX RDF
* License Texts are present multiple times in License Info Doc if generated from spdx report

### CODE REVIEW

**Comment by alexbrdn** [comment]
@bs-jokri, @bodetc, did you notice that travis build failed? please have a look into it,

**Comment by mcjaeger** [comment]
@bs-jokri  two, eh more actually questions:
1) how about changing commits to conventional changelog (and remove the issue numbers)
2) the history does not look OK, rebase instead of merge, ...
4) travis ...
5) make fully qualified issue references in the PR comment?

**Comment by maxhbr** [comment]
Failed tests are: https://travis-ci.org/sw360/sw360portal/jobs/272466917#L3000

**Comment by mcjaeger** [comment]
testing: License names have LicenseRef prefix in License Info Documents when using SPDX -> OK, they have not. (for xhtml and docx as the text variant does have the bug with the missing license names)

File: backend/src/src-licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/outputGenerators/DocxUtils.java [hunk 1]
```diff
@@ -148,27 +148,21 @@ public static void addLicenseTextsHeader(XWPFDocument document, String header) {
...
+            XWPFParagraph licenseParagraph = document.createParagraph();
+            licenseParagraph.setStyle("Heading2");
+            XWPFRun licenseRun = licenseParagraph.createRun();
+            String licenseName = lt.isSetLicenseName() ? lt.getLicenseName() : "Unknown license name";
+            licenseRun.setText(licenseName);
+            addNewLines(licenseRun, 1);
+
+            XWPFParagraph licenseTextParagraph = document.createParagraph();
+            XWPFRun licenseTextRun = licenseTextParagraph.createRun();
+            addFormattedText(licenseTextRun, nullToEmptyString(lt.getLicenseText()), 12);
```
Comment (alexbrdn) [inline backend/src/src-licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/outputGenerators/DocxUtils.java]: magic constant

File: backend/src/src-licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/parsers/SPDXParserTools.java [hunk 2]
```diff
@@ -39,8 +39,10 @@ private static String extractLicenseName(AnyLicenseInfo licenseConcluded) {
         return licenseConcluded.getResource().getLocalName();
     }
 
-    private static String extractLicenseName(ExtractedLicenseInfo extractedLicenseInfo){
-        return ! isNullEmptyOrWhitespace(extractedLicenseInfo.getName()) ? extractedLicenseInfo.getName() : extractedLicenseInfo.getLicenseId();
+    private static String extractLicenseName(ExtractedLicenseInfo extractedLicenseInfo) {
+        return !isNullEmptyOrWhitespace(extractedLicenseInfo.getName())
+                ? extractedLicenseInfo.getName()
+                : extractedLicenseInfo.getLicenseId().replaceAll("LicenseRef-", "");
```
Comment (alexbrdn) [inline backend/src/src-licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/parsers/SPDXParserTools.java]: magic constant

**Comment by bs-jokri** [comment]
@bodetc could you fix that, please.

**Pull request merged**